### PR TITLE
Bootcamp by Jack Paduchowski

### PR DESCRIPTION
**Pull request by jpaduchowski**

### CODE REVIEW

**Comment by ianjfrosst** [comment]
This looks good so far! You've got the probe dropping functions done. But I don't see a command for dropping the probes. Look around in AttitudeManager.c for the relevant code.

File: Autopilot/AttitudeManager/ProbeDrop.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+ * @author Jack Paduchowski
+ * @file ProbeDrop.c
+ * @created September 8, 2017
+ * This file was created for the embedded software bootcamp
+ */
+
+
+#include "PWM.c"
```
Comment (edenfrosst) [inline Autopilot/AttitudeManager/ProbeDrop.c]: You don't need to (and never should) include a .c file.

File: Autopilot/AttitudeManager/AttitudeManager.c [hunk 2]
```diff
@@ -25,6 +25,7 @@
 #include "../Common/Utilities/Logger.h"
 #include "StatusManager.h"
 #include <string.h>
+#include "ProbeDrop.c"
```
Comment (edenfrosst) [inline Autopilot/AttitudeManager/AttitudeManager.c]: You need to make a header for your .c file and include it. Does this code compile? That's a good way to check for errors.

**Comment by ianjfrosst** [comment]
I'll add you to the Asana

**Pull request closed without merging**